.github/workflows/publish-extensions.yml

```diff
@@ -2,9 +2,12 @@ name: Publish Extensions
 on:
   push:
     tags:
-      - "v*"
+      - 'v*'
   workflow_dispatch:
 
+env:
+  SENTRY_DSN_URL: ${{ secrets.SENTRY_DSN_URL }}
```

Comment:
What is the difference b/w DSN and DSN_URL?

Reply:
good spot, this needs to be changed= to the url-less version. I saw in that's what the cli uses implicitly